### ISSUE TITLE
Show which herbs you can pick at farming patchrs

### ISSUE THREAD

**Issue by Gamer1120** [runelite/runelite#5152]
I know this is a mobile screenshot, but it applies to the client as well. It would be nice if it showed "Pick Cadantine" or "Pick Dwarf Weed" instead of "Pick Herbs".

**Comment by deathbeam** [comment]
I mean farming plugin already tells you what you planed

**Comment by Gamer1120** [comment]
True, it's tiny QOL which I don't think it's worth putting a lot of time and effort in yet. Maybe in the future.

**Comment by Abextm** [comment]
Almost all of the data to do this exists in the client, and I don't think it would be too difficult to implement.
